### DISCUSSION TITLE
Secret and ConfigMap hardcoded names

### DISCUSSION THREAD

**Discussion by marioosh11** [argoproj/argo-rollouts#2539] [General]
Hi Argo Lions
Is it possible to avoid hard-coded names for objects below?  We would like to have possibility to change names and use different ones in case when we use it in different helm charts.
```
kind: Secret
metadata:
  name: argo-rollouts-notification-secret
```

and
```
kind: ConfigMap
metadata:
  name: argo-rollouts-notification-configmap
```

BR
Mario

**Comment by illrill** [comment]
If this is related to the Helm chart, I think you want to have the discussion in this repository instead: https://github.com/argoproj/argo-helm.

**Reply by marioosh11** [reply]
Yes you right

Thx

**Reply by marioosh11** [reply]
They answered that its hardcoded in source code see: https://github.com/argoproj/argo-helm/discussions/1800
